### PR TITLE
feat(tooling): adds nvm

### PR DESCRIPTION
**Pull request by evenstensberg**
#6888 

Closing #6956

### Describe your changes:
Adds nvmrc to use same node and npm version throughout the project.

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

**Comment by evenstensberg** [comment]
@unlikelyzero ptal

**Comment by unlikelyzero** [comment]
@evenstensberg please update our readme with the use of `nvm use` where you think it's the most appropriate.

I think maybe here: https://github.com/nasa/openmct/blob/a495e86231346081c58c3770b0b803133e386ee3/README.md?plain=1#L50

**Comment by evenstensberg** [comment]
@unlikelyzero I also added nvm in installation step, ptal

File: README.md [hunk 1]
```diff
@@ -18,11 +18,12 @@ Building and running Open MCT in your local dev environment is very easy. Be sur
 
  `git clone https://github.com/nasa/openmct.git`
 
-2. Install development dependencies. Note: Check the package.json engine for our tested and supported node versions.
+2. Install the correct node version using [nvm](https://github.com/nvm-sh/nvm) (`nvm install`)
```
Comment (unlikelyzero) [inline README.md:21]: I would mention that this is optional

**Review by unlikelyzero — APPROVED** [review]
Can you update this as an optional and it'll be good to go

**Comment by evenstensberg** [comment]
@unlikelyzero  ✅

**Pull request merged**